bindings/nodejs/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// Increased limit required specifically for the call_message method
+#![recursion_limit = "256"]
+#![allow(clippy::needless_borrow)]
+
+mod client;
+mod secret_manager;
+mod wallet;
+
+use iota_sdk_bindings_core::{
+    call_utils_method as rust_call_utils_method, init_logger as rust_init_logger, Response, UtilsMethod,
+};
+use neon::prelude::*;
+use once_cell::sync::Lazy;
+use tokio::runtime::Runtime;
+
+pub static RUNTIME: Lazy<Runtime> = Lazy::new(|| Runtime::new().unwrap());
+
+pub fn init_logger(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+    let config = cx.argument::<JsString>(0)?.value(&mut cx);
+    rust_init_logger(config).expect("failed to init logger");
+    Ok(cx.undefined())
+}
+
+pub fn call_utils_method(mut cx: FunctionContext) -> JsResult<JsString> {
+    let method = cx.argument::<JsString>(0)?.value(&mut cx);
+    let method = match serde_json::from_str::<UtilsMethod>(&method) {
+        Ok(method) => method,
+        Err(err) => {
+            return Ok(cx.string(
+                serde_json::to_string(&Response::Error(err.into()))
+                    .expect("the response is generated manually, so unwrap is safe."),
```

Comment:
that's a really nice message to read should this ever panic due to cosmic rays or something.

On a more serious note: this should probably be turned into a comment and just an `unwrap` or a `expect("json to string error")

Reply:
Updated